runtime/trappist/src/lib.rs

```diff
@@ -654,13 +656,66 @@ impl pallet_withdraw_teleport::Config for Runtime {
 	type WeightInfo = weights::pallet_withdraw_teleport::WeightInfo<Runtime>;
 }
 
-impl pallet_lockdown_mode::Config for Runtime {
+/// Calls that can bypass the safe-mode pallet.
+pub struct SafeModeWhitelistedCalls;
+impl Contains<RuntimeCall> for SafeModeWhitelistedCalls {
+	fn contains(call: &RuntimeCall) -> bool {
+		match call {
+			RuntimeCall::System(_)
+			| RuntimeCall::SafeMode(_)
+			| RuntimeCall::TxPause(_)
+			| RuntimeCall::Balances(_) => true,
+			_ => false,
+		}
+	}
+}
+
+parameter_types! {
+	pub const EnterDuration: BlockNumber = 4 * HOURS;
+	pub const EnterDepositAmount: Balance = 2_000_000 * DOLLARS;
+	pub const ExtendDuration: BlockNumber = 2 * HOURS;
+	pub const ExtendDepositAmount: Balance = 1_000_000 * DOLLARS;
+	pub const ReleaseDelay: u32 = 2 * DAYS;
+}
+
+impl pallet_safe_mode::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
-	type LockdownModeOrigin = frame_system::EnsureRoot<Self::AccountId>;
-	type BlackListedCalls = RuntimeBlackListedCalls;
-	type LockdownDmpHandler = LockdownDmpHandler;
-	type XcmExecutorManager = XcmExecutionManager;
-	type WeightInfo = weights::pallet_lockdown_mode::WeightInfo<Runtime>;
+	type Currency = Balances;
+	type RuntimeHoldReason = RuntimeHoldReason;
+	type WhitelistedCalls = SafeModeWhitelistedCalls;
+	type EnterDuration = EnterDuration;
+	type ExtendDuration = ExtendDuration;
+	type EnterDepositAmount = EnterDepositAmount;
+	type ExtendDepositAmount = ExtendDepositAmount;
```

Comment:
The issue mentioned "_Permissionless operations (enter, extend) should be disabled._". IIRC it's possible to disallow the permissionless operations by specifying a zero amount for these params ? Please verify.

Reply:
Very true, I didn't see that. Fixed in [33b1736](https://github.com/paritytech/trappist/pull/305/commits/33b173617174bd419a5246670ac3699375c25d51).